core/runtime/bindings.rs

```diff
@@ -323,6 +323,17 @@ pub(crate) fn initialize_deno_core_ops_bindings<'s>(
   }
 }
 
+pub fn register_op_method(
```

Comment:
We should try to create these once when the JsRuntime is created and then look them up.